concent_api/core/tests/test_unit_admin_tags.py

```diff
@@ -0,0 +1,150 @@
+import datetime
+import hashlib
+from assertpy import assert_that
+from freezegun import freeze_time
+from golem_messages.message.concents import ForceGetTaskResult
+from golem_messages.factories.tasks import ReportComputedTaskFactory
+from common.helpers import get_current_utc_timestamp
+from core.message_handlers import store_subtask
+from core.models import Subtask
+from core.templatetags.admin_tags import get_longest_lasting_subtask_timestamp, get_time_until_concent_can_be_shut_down
+from core.tests.utils import ConcentIntegrationTestCase
+from core.utils import hex_to_bytes_convert
+from api_testing_common import create_signed_task_to_compute
+from api_testing_common import timestamp_to_isoformat
+
+
+def store_report_computed_task_as_subtask(current_time, task_id, subtask_id, deadline, next_deadline, subtask_state):
+    file_content = '1'
+    file_size = len(file_content)
+    file_check_sum = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
+    with freeze_time(timestamp_to_isoformat(current_time - 100)):
```

Comment:
Maybe use some concent constant instead of `-100` in this place. If in the future times will change your test may fail. I had the same problem in one of my last pull requests :)

Reply:
Done.